src/ApparentHorizons/TagsDeclarations.hpp

```diff
@@ -0,0 +1,76 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstdint>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/TMPL.hpp"
+
+class DataVector;
+
+namespace StrahlkorperTags {
+
+namespace StrahlkorperTags_detail {
```

Comment:
Are these details needed in the declarations file, or can they be moved to Tags.hpp?

Reply:
Here's why I did this. Before I made this change, I had the following problem:

 - StrahlkorperGr.hpp depends on Tags.hpp  because it uses type aliases in StrahlkorperTags_detail (defined in Tags.hpp)

 - I want to add ComputeItems to Tags.hpp, which will make Tags.hpp depend on StrahlkorperGr.hpp (which declares the functions used by those ComputeItems).

So how to break the circular dependency?  A few options:

1.  Put `StrahlkorperTags_detail` into its own hpp file, so it can be included into Tags.hpp and StrahlkorperGr.hpp.

1.  Same as previous but also put the Tags.hpp declarations into the same hpp file with `Strahlkorper_tags_detail`, since at some point we will want only the declarations and not the definitions.  This is what I did (but I see now why it is confusing).

1. Have three different files: Tags.hpp, TagsDeclarations.hpp, and TagsTypeAliases.hpp, where the last file includes what is now in `StrahlkorperTags_detail` (and put into a namespace without 'detail' in the name).

1. Remove the usage of `Strahlkorper_tags_detail` in StrahlkorperGr.hpp by explicitly specifying the long types instead of using type aliases.  Then `Strahlkorper_tags_detail` is used only in Tags.hpp.  But we need to write out four-line type names in StrahlkorperGr.hpp.

Preferences? Option 3 seems cleanest to me. (or some other suggestion)?